oaknational/Oak-Web-Application
# Extract out helpers from pages to fix bundle issue

## Description
Extracted out curriculum API helpers to fix OWA incorrectly bundling curriculum-api-2023 graphql files.

## Issue(s)

Fixes `CUR-1115`

## How to test
This should be a no-op from a user experience point of view so checking for regressions across the visualiser and docx 

1. Go to https://deploy-preview-3026--oak-web-application.netlify.thenational.academy
2. Check for regressions, nothing though have changed

## Screenshots
Before:
<img width="834" alt="Screenshot 2024-11-26 at 13 35 06" src="https://github.com/user-attachments/assets/ef3299a2-e275-471f-9495-0cc8ac259d78">

After:
<img width="834" alt="Screenshot 2024-11-26 at 13 34 01" src="https://github.com/user-attachments/assets/e3997841-1b17-4782-87b5-479cd19edb2d">
